src/classes/RD2_ElevateIntegrationService_TEST.cls

```diff
@@ -129,6 +129,85 @@ private class RD2_ElevateIntegrationService_TEST {
 
     }
 
+        /***
+    * @description Verifies that any new opportunities are linked to a existing RD if the CommitmentId matches
+    */
+    @isTest
+    private static void shouldLinkOppsToRDMatchingCommitmentIdOnOppInsert() {
```

Comment:
- Since there are five ACs in the WI, and the second test below verifies the AC4, do we have other ACs covered in tests? 
- It seems this test is covering more than one AC or concept, let's write it so there is one test per concept.